06_puppy/alextmz/storer_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type storerSuite struct {
+	suite.Suite
+	p  []Puppy
+	st Storer
+}
+
+func (su *storerSuite) SetupTest() {
+	su.p = nil
+	var p = []Puppy{
+		{ID: 256, Breed: "Dogo", Colour: "White", Value: 500},
+		{ID: 512, Breed: "Mastiff", Colour: "Brindle", Value: 700},
+		{ID: 768, Breed: "Fila", Colour: "Golden", Value: 900},
+	}
+	su.p = append(su.p, p...)
+	err := su.st.CreatePuppy(&su.p[0])
+	su.Nil(err)
```

Comment:
@alex if this Create function fails the setup would fail consider using suite.FailNow ?

Reply:
Yes, that makes a lot more sense, thanks!